web/src/features/prompts/server/routers/promptRouter.ts

```diff
@@ -775,6 +796,24 @@ export const promptRouter = createTRPCRouter({
 
         // Unlock cache
         await promptService.unlockCache({ projectId, promptName });
+
+        // Trigger webhooks for prompt label update
+        const updatedPrompts = await ctx.prisma.prompt.findMany({
+          where: {
+            id: { in: touchedPromptIds },
+            projectId,
+          },
+        });
+
+        // Send webhooks for ALL affected prompts
+        await Promise.all(
+          updatedPrompts.map(async (prompt) =>
+            promptChangeEventSourcing(
+              await promptService.resolvePrompt(prompt),
+              "updated",
+            ),
```

Comment:
Is there a prisma hook that we can use? Maybe that would cover audit logs and event sourcing.